source/index.txt

```diff
@@ -19,46 +19,83 @@ MongoDB Go Driver
 Introduction
 ------------
 
-The official MongoDB Go Driver enables Go applications to connect to
-MongoDB and work with data.
+The official MongoDB Go Driver enables Go applications to connect to and
+work with data in MongoDB. This page describes each section of the
+driver documentation. 
+
+Learn
+-----
+
+Visit the Developer Hub for content, resources, community, and social
+engagement for developers related to the Go driver:
+
+- To view articles on Go, see the `How To's and Articles page <https://www.mongodb.com/developer/learn/?content=Articles&text=Go>`__.
+- To view the forum on Go, see the `forums page <https://www.mongodb.com/community/forums/tag/golang>`__.
+
+Quick Start
+-----------
+
+To get up and running quickly, take a look at the :doc:`Quick Start
+</quick-start>` to learn how to establish a connection to MongoDB Atlas
+and begin to work with data.
 
 Usage Examples
 --------------
+
 The :doc:`Usage Examples </usage-examples>` section provides
 runnable code snippets and explanations for common methods. We recommend
 this section for users that are new to the MongoDB Go Driver.
 
 Fundamentals
 ------------
+
 Whether you are new to MongoDB or you just want to brush up on the core
-concepts, you can check out the **Fundamentals** section which contains
-learning material on using the Go driver for the following:
+concepts, you can check out the **Fundamentals** section which covers
+the following material in the Go driver:
 
-* TODO
+- :doc:`Connect to MongoDB </fundamentals/connection>`
+- :doc:`Context Usage in MongoDB </fundamentals/context>`
+- :doc:`Authenticate with MongoDB </fundamentals/auth>`
+
+.. - :doc:`Specify an API Version </fundamentals/versioned-api>`
+.. - :doc:`Use the Driver's Data Formats </fundamentals/data-formats>`
+.. - :doc:`Perform CRUD Operations </fundamentals/crud>`
+.. - :doc:`Perform Aggregations </fundamentals/aggregation>`
+.. - :doc:`Construct Indexes </fundamentals/indexes>`
+.. - :doc:`Specify Collations </fundamentals/collations>`
+.. - :doc:`Record Events in the Driver </fundamentals/logging>`
+.. - :doc:`Use Driver Events in your Code </fundamentals/monitoring>`
+.. - :doc:`Store and Retrieve Files in MongoDB </fundamentals/gridfs>`
 
 API
 ---
-See the `API Documentation <https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo>`_ link to api docs if you are looking
-for technical information about classes, methods, and configuration objects
-within the MongoDB Go Driver.
 
-FAQ
----
-The :doc:`Frequently Asked Questions (FAQ) </faq>` section provides answers
-to commonly asked questions about the MongoDB Go Driver.
+See the `API Documentation
+<https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo>`__ if you are
+looking for technical information about classes, methods, and
+configuration objects within the MongoDB Go Driver.
```

Comment:
I think Go calls them configuration structs, but that could be too jargon-y for the landing page. is there another way to describe this? Types and functions?

Reply:
Agreed! Types and functions sounds the most accurate.